share/eds/accessor.go

```diff
@@ -26,7 +26,7 @@ type Accessor interface {
 	// choose which axis to use for proof. Chosen axis for proof should be indicated in the returned
 	// Sample.
 	// TODO(@Wondertan): change to SampleIndex
-	Sample(ctx context.Context, rowIdx, colIdx int) (shwap.Sample, error)
+	Sample(ctx context.Context, rowIdx, colIdx int, sampleIdx shwap.SampleIndex) (shwap.Sample, error)
```

Comment:
Btw, the idea is to use SampleIndex instead of rowIdx, colIdx

Reply:
So, drop these 2 right?

Reply:
The SampleIndex has Coordinates method that gives you the equivalent

Reply:
Done in most places. Something should be updated in tests.